src/storage/db_storage.hpp

```diff
@@ -195,6 +202,11 @@ struct DbStorage {
   std::map<trx_hash_t, TransactionStatus> getAllTransactionStatus();
 
   // PBFT manager
+  uint64_t getPbftMgrPreviousRoundStatus(PbftMgrPreviousRoundStatus const& field);
+  void savePbftMgrPreviousRoundStatus(PbftMgrPreviousRoundStatus const& field, uint64_t const& value);
+  void addPbftMgrPreviousRoundStatus(PbftMgrPreviousRoundStatus const& field, uint64_t const& value,
```

Comment:
reference to the enum makes no sense. You are using reference (which is 4 bytes big) to the enum (which should be of type uint8_t). Just use "void addPbftMgrPreviousRoundStatus(PbftMgrPreviousRoundStatus field, ...)

Reply:
Fixed, thanks